packages/swirl-components/src/components/flip-description-list-item/flip-description-list-item.css

```diff
@@ -0,0 +1,67 @@
+@import "../../styles/media-queries.css";
+
+:host {
+  display: block;
+
+  & * {
+    box-sizing: border-box;
+  }
+}
+
+.description-list-item {
+  display: block;
+  max-width: 100%;
+  color: var(--s-text-default);
+  line-height: var(--s-line-height-base);
+}
+
+.description-list-item__term {
+  display: block;
+  margin-bottom: var(--s-space-4);
+  font-weight: var(--s-font-weight-semibold);
+}
+
+.description-list-item__description {
+  display: block;
+  max-width: 40rem;
+  margin: 0;
+}
+
+@container description-list (min-width: 480px) {
+  .description-list-item {
+    display: flex;
+  }
+
+  .description-list-item__term {
+    margin-right: var(--s-space-16);
+    margin-bottom: 0;
+    flex-basis: 12rem;
+    flex-grow: 0;
+    flex-shrink: 0;
+  }
+
+  .description-list-item__description {
+    flex-grow: 1;
+  }
+}
+
+/* Fallback styles for browsers without container query support */
+@supports not (container-type: inline-size) {
+  @media (--from-tablet) {
+    .description-list-item {
+      display: flex;
```

Comment:
when i look at the design i would argue that we should use <code>justify-content: space-between</code> here?
![image](https://user-images.githubusercontent.com/23700195/189837830-56ddfe7f-2d50-4829-b5c4-dda63f197918.png)

Reply:
at least this looks *"correcter"* to me ^^
![image](https://user-images.githubusercontent.com/23700195/189837973-6c47446d-7d00-401d-b7c5-6430c86b7bb9.png)

Reply:
I would argue that this creates too much white space between the term and the description on wide screens, doesn't it?